WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -226,9 +228,15 @@ public boolean onOptionsItemSelected(final MenuItem item) {
 
             if (peopleInviteFragment == null) {
                 Blog blog = WordPress.getCurrentBlog();
-                peopleInviteFragment = PeopleInviteFragment.newInstance(blog.getDotComBlogId());
+                String dotComBlogId = blog.getDotComBlogId();
+                if (dotComBlogId != null && !dotComBlogId.isEmpty()) {
```

Comment:
We could probably use  `if(!TextUtils.isEmpty(dotComBlogId))` here to simplify it a little bit.